base_local_planner/src/base_local_planner/plot_scored_sampling_planner_costs.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python
```

Comment:
I put it here because it is just useful for the local planner.  I could move it to tools, since that is where all of the plotting stuff lives.  Probably makes more sense there.